packages/hash/api/src/graph/workspace-types.ts

```diff
@@ -22,12 +26,22 @@ export let WORKSPACE_TYPES: {
     orgName: PropertyTypeModel;
     orgSize: PropertyTypeModel;
     orgProvidedInfo: PropertyTypeModel;
+
+    // OrgMembership-related
+    responsibility: PropertyTypeModel;
```

Comment:
From just looking at the variable, it's unclear, what `responsibility` means, who/what is responsible for what/who?

Reply:
The variable name is just the title of the property type. And the property type will have a better description (in the long run probably, I don't know if we've written any right now).

I think this is probably fine, although I don't really know what responsibility is either, maybe it's something like "admin" or "maintainer"

Reply:
To get us on the same page, the origin of this property is we want to capture somewhere what a user's "role" at an organization is. We renamed this to responsibility in the workspace API so that it wouldn't be confused with any permissioning "roles", which is not what this is. Possible values would be things like "Marketing", "Sales", etc - depending what the user specifies.

<img width="429" alt="image" src="https://user-images.githubusercontent.com/42802102/190615096-7f836356-8062-40cb-b3c5-8fe71122c15b.png">

Reply:
> We should have at least a small comment describing

Instead of a comment, I think it'd be better for the description to be contained in the property type `description` field instead? That way it's actually captured in the type, and not just our code.

I've added this in https://github.com/hashintel/hash/pull/1068/commits/13e57f2d6b4a055b9b78023467b89b76906eb435, let me know what you think.

Reply:
This is really useful, thanks!